vitest.config.ts

```diff
@@ -34,6 +22,47 @@ export default defineConfig({
         'src/utils/http-status.ts',
       ],
     },
-    pool: 'forks',
+    projects: [
+      './runtime-tests/*/vitest.config.ts',
+      {
+        esbuild: {
+          jsx: 'automatic',
+          jsxImportSource: './src/jsx',
+        },
+        extends: true,
+        test: {
+          environment: 'node',
+          exclude: [...configDefaults.exclude, 'src/jsx', '**/*.case.test.*'],
+          include: [
+            'src/**/(*.)+(spec|test).+(ts|tsx|js)',
+            'scripts/**/(*.)+(spec|test).+(ts|tsx|js)',
+            'build/**/(*.)+(spec|test).+(ts|tsx|js)',
+          ],
+          name: 'unit',
+        },
+      },
+      {
+        esbuild: {
+          jsx: 'automatic',
+          jsxImportSource: './src/jsx',
+        },
+        extends: true,
+        test: {
+          include: ['src/jsx/dom/**/(*.)+(spec|test).+(ts|tsx|js)', 'src/jsx/hooks/dom.test.tsx'],
+          name: 'jsx-runtime-default',
+        },
+      },
```

Comment:
This runs the same tests as what `vitest -c .vitest.config/jsx-runtime-default.ts --run` previously ran